Update dbt-support.md (#6908)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-support.md

The global dbt Support team is available to dbt Cloud customers by [email](mailto:support@getdbt.com) or by clicking **Create a support ticket** through the dbt Cloud navigation.

> ### Ask dbt Support AI widget

Author: ```suggestion
### Ask Support Assistant
```

> Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud useres and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 

Author: ```suggestion
Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud users and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 
```

> Ask dbt Support is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud useres and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 

Author: ```suggestion
Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud useres and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 
```

> dbt Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud users and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 

Author: ```suggestion
dbt Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud users and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 
```

> dbt Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud users and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 

Reviewer: Quick explanations is a little vague. Is it like feature summaries?
```suggestion
dbt Support Assistant is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud users and can help answer troubleshooting questions, give a synopsis of features and functionality, or link to relevant documentation. 
```

> The Ask dbt Support widget is an AI widget that provides instant, AI-generated responses to common questions. This feature is available to dbt Cloud useres and can help answer troubleshooting questions, provide quick explanations, or link to relevant documentation. 

Author: is this the 'official name'?

> The Support Assistant AI widget is separate from the [dbt Copilot feature](/docs/cloud/dbt-copilot). dbt Copilot helps with model building and query generation, whereas Ask dbt Support focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect Ask dbt Support's availability.

Author: ```suggestion
The Support Assistant AI widget is separate from [dbt Copilot](/docs/cloud/dbt-copilot), a powerful AI-engine that helps with code generation to accelerate your analytics workflows. Support Assistant focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect Support Assistant's availability.
```

> The Ask dbt Support AI widget is separate from the [dbt Copilot feature](/docs/cloud/dbt-copilot). dbt Copilot helps with model building and query generation, whereas Ask dbt Support focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect Ask dbt Support's availability.

Author: ```suggestion
The Support Assistant AI widget is separate from the [dbt Copilot feature](/docs/cloud/dbt-copilot). dbt Copilot helps with model building and query generation, whereas Ask dbt Support focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect Ask dbt Support's availability.
```

> The dbt Support Assistant AI widget is separate from [dbt Copilot](/docs/cloud/dbt-copilot), a powerful AI-engine that helps with code generation to accelerate your analytics workflows. dbt Support Assistant focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect dbt Support Assistant's availability.

Reviewer: ```suggestion
The dbt Support Assistant AI widget is separate from [dbt Copilot](/docs/cloud/dbt-copilot), a powerful AI engine that helps with code generation to accelerate your analytics workflows. The dbt Support Assistant focuses on answering documentation and troubleshooting-related questions. Enabling or disabling AI features in dbt Cloud won't affect the dbt Support Assistant's availability.
```

:::info

> Note that AI-generated responses may not always be accurate. If you receive unexpected or incorrect information, we recommend verifying details in our documentation or [reaching](mailto:support@getdbt.com) out to the Support team.

Reviewer: Recommend a slightly more positive approach
```suggestion
We recommend validating information received in AI responses for any scenario using our documentation. Please [contact support](mailto:support@getdbt.com) to report incorrect information provided by the Support Assistant.
```
